source/mainnet/net/browser-wallet/browser-wallet-faq.rst

```diff
@@ -0,0 +1,81 @@
+
+    As long as you have your secret recovery phrase, you have your CCDs. If you break a phone containing your wallet, you haven’t lost your CCDs. You can simply enter your secret recovery phrase into a newly downloaded |mw-gen2| or |bw|.
+
+.. dropdown:: How can I keep my secret recovery phrase secure?
+
+    - Don't take a picture of it; if you save photos to any cloud provider this could potentially expose your secret recovery phrase so anyone could access your accounts and funds.
+    - Don't keep it with your device. If you lose your device, anyone who finds it could access your accounts and funds.
+    - Put it in a safe location. Keep your secret recovery phrase in a safe location that is fireproof and waterproof, and that you will remember and can access relatively easily. There are companies that make devices, such as https://shop.ledger.com/products/the-billfodl that can safely store your secret recovery phrase.
+    - Keep multiple physical copies of your secret recovery phrase in safe locations.
+
+.. dropdown:: Do I still need to make backups of my wallet?
+
+    No. For the |bw| you do not need to make backups. Your secret recovery phrase that you write down is the only way to recover your accounts and identities.
+
+.. dropdown:: Can I migrate from the |mw-gen1| to the |bw|?
+
+    No. Because the way that keys are protected differs between the old and new wallets you cannot simply migrate. If you use the |mw-gen1| but want to use the |bw|, you should do the following:
+
+    #. Download the |bw| and set it up so you have a secret recovery phrase, a verified identity, and at least one account.
+    #. Open the |mw-gen1| and send your funds from it to your new account(s) in the |bw|.
+    #. Once you are sure that all of your funds have been transferred and you have no incoming transfers, you can delete the |mw-gen1| on your phone. You can also keep the |mw-gen1| and use both wallets. Just make sure that you continue to make :ref:`backups<export-import>` if you continue using the |mw-gen1|.
+
+    Note that in the near future identity and account creation will be disabled in |mw-gen1|.
+
+.. dropdown:: Can I access my wallet on multiple devices with the secret recovery phrase?
+
+    Yes, you can access your wallet using |mw-gen2| and |bw|. You can recover your wallet in a device that uses either of these. Be aware that any names you have given to identities and accounts are **specific to the device**, so if you have used special names for them, they will not appear when you recover the wallet on another device. You can edit the account name and edit the identity name, if desired. Also, only addresses for your own account are added to the address book. Other addresses cannot be recovered. So you must add those manually.
```

Comment:
```suggestion
    Yes, you can access your wallet using |mw-gen2| and |bw|. You can recover your wallet in a device that uses either of these. Be aware that any names you have given to identities and accounts are **specific to the device**, so if you have used special names for them, they will not appear when you recover the wallet on another device. You can edit the account name and edit the identity name, if desired.
```
The browser wallet does not have an address book (yet?).
And you can't edit the account names in the initial version. (but that will be added soon, I think)